### PR TITLE
Tempo: Remove Loki tab

### PR DESCRIPTION
**Pull request by joey-grafana**
**What is this feature?**

Removes the Loki tab from Tempo.

**Why do we need this feature?**

This tab has been around since before we could natively query the Tempo db for traces, and is used by a very low number of users in comparison to the search or TraceQL tab. If users would like to see what logs are linked to a specific service they can use the trace to logs feature which provides an easy way to create a custom link and set an appropriate time range if necessary.

**Who is this feature for?**

Tempo users.

**Special notes for your reviewer:**

Screenshot 2024-03-06 at 09 14 38

# Release notice breaking change

We've removed the Loki tab from the Tempo data source. You can still access Logs through the Loki data source or can also create a link from Tempo to Loki via our [trace to logs](https://grafana.com/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#trace-to-logs) feature.

### CODE REVIEW

**Review by jdbaldry — APPROVED** [review]
docs removals look consistent with the changes. 

Adding @knylander-grafana as well since it touches the Tempo data source.

**Review by adrapereira — APPROVED** [review]
Beautiful!

File: public/app/plugins/datasource/tempo/dataquery.gen.ts [hunk 1]
```diff
@@ -68,9 +68,9 @@ export const defaultTempoQuery: Partial<TempoQuery> = {
 };
 
 /**
- * search = Loki search, nativeSearch = Tempo search for backwards compatibility
+ * nativeSearch = Tempo search for backwards compatibility
```
Comment (fabrizio-grafana) [inline public/app/plugins/datasource/tempo/dataquery.gen.ts:71]: Is the comment correct? I think with the removal of the old search tab, we don't have any backward compatibility issues.

**Review by fabrizio-grafana — APPROVED** [review]

File: public/app/plugins/datasource/tempo/dataquery.gen.ts (hunk 1 above)
Comment (joey-grafana) [reply public/app/plugins/datasource/tempo/dataquery.gen.ts:71]: Yeah I think you're right, will remove this comment in the PR that removes the old search.

**Pull request merged**